alpha/lib/model/LiveEntry.php

```diff
@@ -609,19 +613,28 @@ private function storeInCache($key)
 			KalturaLog::debug("cacheStore is null. cacheType: $cacheType . returning false");
 			return false;
 		}
-		return $cacheStore->set($key, true, kConf::get('media_server_cache_expiry', 'local', self::DEFAULT_CACHE_EXPIRY));
+		KalturaLog::debug("Set cache key [$key] from store [$cacheType] ");
+		$returnValue =  $cacheStore->set($key, true, kConf::get('media_server_cache_expiry', 'local', self::DEFAULT_CACHE_EXPIRY));
+		$ans = $cacheStore->get($key);
```

Comment:
You never return the $returnValue.

Remove the:
$ans = $cacheStore->get($key); and the debug it duplicate to the previous debug.